src/test-basic.c

```diff
@@ -196,10 +196,57 @@ static void test_dbus_body(void) {
         free(data);
 }
 
+static void test_skip_fixed_size_array(void) {
+        static const CDVarType types[] = {
+                /* ayi */
+                C_DVAR_T_INIT(
+                        C_DVAR_T_ARRAY(
+                                C_DVAR_T_y
+                        )
+                ),
+                C_DVAR_T_INIT(C_DVAR_T_i)
+        };
+        _c_cleanup_(c_dvar_freep) CDVar *var = NULL;
+        size_t n_data;
+        void *data;
+        int r;
+        unsigned i;
+
+        /*
+         * Simple test that marshals a fixed-size array, with additional integer added after the array.
+         */
+
+        r = c_dvar_new(&var);
+        c_assert(!r);
+
+        c_dvar_begin_write(var, (__BYTE_ORDER == __BIG_ENDIAN), types, 2);
+        c_dvar_write(var, "[");
+        for (i = 0; i < 10000001; i++)
+                c_dvar_write(var, "y", i % 256);
```

Comment:
`meson` has special support for benchmarks. I would like to keep unit-tests small, so we can execute them in dependent projects without any problems. If we really want a benchmark, I would convert this into a benchmark. Though I think it would be fine to skip this and just keep some more elaborate unit-tests for `c_dvar_skip()`.